spec/models/membership_spec.rb

```diff
@@ -132,12 +138,26 @@
     end
   end
 
-  context 'locked group' do
+  context 'locked membership' do
     it 'cannot be destroyed' do
+      group = FactoryGirl.create(:finalizing_group)
+      membership = group.memberships.first
+      expect { membership.destroy }.not_to change { group.memberships_count }
+    end
+    it 'must be accepted' do # rubocop:disable RSpec/ExampleLength
+      group = FactoryGirl.create(:finalizing_group)
+      membership = Membership.new(group: group,
+                                  user: FactoryGirl.create(:student,
+                                                           draw: group.draw),
+                                  status: 'requested')
+      membership.locked = true
+      expect(membership).not_to be_valid
+    end
```

Comment:
You don't need to re-enable unless it was on it's own line - the way it's written above it only suppresses the error on the that specific line.